javascript/frameworks/cap/test/queries/cqlinjection/srv/service1.js

```diff
@@ -0,0 +1,786 @@
+const cds = require("@sap/cds");
+
+module.exports = class Service1 extends cds.ApplicationService {
+  init() {
+    /* ========== 1. Service1 running query on the database service using `cds.run` and friends using Fluent API ========== */
+    this.on("send00111", async (req) => {
+      const { id } = req.data;
+      const query = SELECT.from`Entity1`.where("ID=" + id);
+      cds.run(query);
+    });
+
+    this.on("send00112", async (req) => {
+      const { id } = req.data;
+      const query = SELECT.from`Entity1`.where(`ID=` + id);
+      cds.run(query);
+    });
+
+    this.on("send00113", async (req) => {
+      const { id } = req.data;
+      const query = SELECT.from`Entity1`.where(`ID=${id}`);
+      cds.run(query);
+    });
+
+    this.on("send00114", async (req) => {
+      const { id } = req.data;
+      const query = SELECT.from`Entity1`.where`ID=${id}`;
+      cds.run(query); // FP
```

Comment:
No longer an FP?

Reply:
Same strategy as [this comment](https://github.com/advanced-security/codeql-sap-js/pull/200#discussion_r2178535082) will be used and will be addressed together.